crypto/ct/ct_sct_ctx.c

```diff
@@ -111,7 +111,7 @@ static int ct_x509_get_ext(X509 *cert, int nid, int *is_duplicated)
  * AKID from the presigner certificate, if necessary.
  * Returns 1 on success, 0 otherwise.
  */
-static int ct_x509_cert_fixup(X509 *cert, X509 *presigner)
+__owur static int ct_x509_cert_fixup(X509 *cert, X509 *presigner)
```

Comment:
Many of these parameters that aren't const, I tried to change to const but found that I ran into problems with X509/ASN.1 functions not taking const parameters when they should. In those cases, should I take the parameters as const here and cast away the constness at those call sites?